advisories/github-reviewed/2024/08/GHSA-88g2-r9rw-g55h/GHSA-88g2-r9rw-g55h.json

```diff
@@ -12,10 +12,6 @@
     {
       "type": "CVSS_V3",
       "score": "CVSS:3.1/AV:L/AC:H/PR:N/UI:R/S:U/C:N/I:L/A:N"
-    },
-    {
-      "type": "CVSS_V4",
-      "score": "CVSS:4.0/AV:N/AC:L/AT:P/PR:N/UI:P/VC:N/VI:L/VA:N/SC:N/SI:N/SA:N"
     }
```

Comment:
I don't know why this sometimes happens when advisories are edited; I only edited the `gitoxide-core` affected version metadata and nothing else. But I've seen this kind of thing happen before and I'm guessing it's okay. If not, or otherwise if there's anything I should do related to this, please let me know.

Reply:
Ya this is fine. It's just a bug in how the improvement form handles CVSS4 strings at the moment. We've got it in an issue, but unclear when we'll get this resolved. Many thanks for the PR though 👍

Reply:
That's great--I'm glad you're aware of the issue, and thanks for the super-quick reply!

Reply:
np, happy to help 😄